Source/WebCore/platform/graphics/cg/IntRectCG.cpp

```diff
@@ -39,11 +39,11 @@ IntRect::operator CGRect() const
 
 IntRect enclosingIntRect(const CGRect& rect)
 {
-    int l = static_cast<int>(floorf(rect.origin.x));
-    int t = static_cast<int>(floorf(rect.origin.y));
-    int r = static_cast<int>(ceilf(CGRectGetMaxX(rect)));
-    int b = static_cast<int>(ceilf(CGRectGetMaxY(rect)));
-    return IntRect(l, t, r - l, b - t);
+    int L = static_cast<int>(std::floor(rect.origin.x));
```

Comment:
Why are we capitalizing these? I don't think we should.

Reply:
style complains otherwise but yeah I will change back.